frontend/pages/DetailedDataset/[id].js

```diff
@@ -308,14 +308,14 @@ export default function DetailedDataset({ data, uri, prevUserId, prevLoggedUsern
         <Divider variant="fullWidth" />
         <h3 style={{ fontWeight: '600' }}>Datasettet blir brukt til:</h3>
         {Object.values(data.subscriptions).length == 0 ? (
-          <>
+          <div>
```

Comment:
Should define what kind of text this is (ex with Typography) so it is equal on all plattforms and browsers.